cmd/notation/verify.go

```diff
@@ -35,6 +35,19 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "verify [reference]",
 		Short: "Verifies OCI Artifacts",
+		Long: `
+** This command is in preview and under development. **
+Prerequiste: a trusted certificate needs to be generated using command "notation cert". 
```

Comment:
`notation cert` doesn't generate certificates except generating self-signed certificates. For alpha.4, `notation cert ` manages certificate list for verification.

Reply:
Update to `a trusted certificate needs to be added or generated using the command "notation cert"`.